dashboard/app/assets/javascripts/dashboard.controller.js

```diff
@@ -0,0 +1,49 @@
+(function() {
+  'use strict';
+
+  angular.module('qat-dashboard').controller('DashboardController', [
+    'testExecutor',
+    function(testExecutor) {
+      var vm = this;
```

Comment:
@christianvuerings To add to my note above, I was following this styleguide: https://github.com/johnpapa/angular-styleguide. Specifically, the following sections:
- https://github.com/johnpapa/angular-styleguide#controlleras-with-vm
- https://github.com/johnpapa/angular-styleguide#bindable-members-up-top

`var vm = this;` allows us to access `this` inside a function. Binding functions at the top (before defining helps with readability. )

I have since found, that I don't have to access this anywhere within functions if I am good with my variable declarations. And I can preserve the instanceof behavior by using the following pattern:
```
angular.extend(this, {
  testExecsMap: testExecutor.all,
  // ...
})
```

I will move the code to this pattern.